src/lib.rs

```diff
@@ -66,6 +70,18 @@ pub fn load_and_index_pois(
     let es = args.es.clone();
     let langs = &args.langs;
     let rubber = &mut mimir::rubber::Rubber::new(&es);
+
+    let poi_creation_date = get_index_creation_date(rubber, &format!("{}_poi", &args.dataset));
+    let addr_creation_date = get_index_creation_date(rubber, &format!("{}_addr", &args.dataset));
```

Comment:
A similar problem here with index names.
Also note that mimir maintains a default alias for each type, so the dataset name is not required to fetch the state of the addr index.
```suggestion
    let poi_creation_date = get_index_creation_date(rubber, &format!("{}_poi_{}", PREFIX, &args.dataset));
    let addr_creation_date = get_index_creation_date(rubber, &format!("{}_addr", PREFIX));
```

Reply:
It appears that the dataset name is not required either for POIs?

```suggestion
    let poi_creation_date = get_index_creation_date(rubber, &format!("{}_poi", PREFIX));
    let addr_creation_date = get_index_creation_date(rubber, &format!("{}_addr", PREFIX));
```

Querying `https://${ES_ADDR}/_cat/indices/munin_poi?h=creation.date` I actually get expected result \o/

Other issues should also be addressed in f2700ef